docs/architecture.md

```
@@ -42,17 +41,19 @@ This architecture ensures:
 | --- | --- |
 | `/docker` | Docker-related files for runtime container |
 | `/docs` | Documentation files and guides |
+| `/agents` | Agent profiles (prompts, tools, extensions per profile) |
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)